README.md

```diff
@@ -112,19 +69,30 @@ If you are using Spectral in your project and want to be listed in the examples
 
 ## Helpful Links
 
-- [JSONPath Online Evaluator](http://jsonpath.com/), a helpful tool to determine what `path` you want
+- [JSONPath Online Evaluator](http://jsonpath.com/), a helpful tool to determine what `given` path you want
 - [stoplightio/json](https://github.com/stoplightio/json), a library of useful functions for when working with JSON
 - [stoplightio/yaml](https://github.com/stoplightio/yaml), a library of useful functions for when working with YAML, including parsing YAML into JSON, and a few helper functions such as `getJsonPathForPosition` or `getLocationForJsonPath`
 
 ## Thanks :)
 
 - [Phil Sturgeon](https://github.com/philsturgeon) for collaboration and creating Speccy
-- [Mike Ralphson](https://github.com/MikeRalphson) for kicking off the Spectral CLI
+- [Mike Ralphson](https://github.com/MikeRalphson) for kicking off the Spectral CLI and his work on Speccy
 
 ## Support
 
-If you have a bug or feature request, please open an issue [here](https://github.com/stoplightio/spectral/issues).
+If you have a bug or feature request, please [create an issue](https://github.com/stoplightio/spectral/issues).
 
 If you need help using Spectral or have a support question, please use the [Stoplight Community forum](https://community.stoplight.io). We've created an open source category for these questions. It's also a great place to share your implementations.
 
 If you want to discuss something in private, you can reach out to Stoplight support at [support@stoplight.io](mailto:support@stoplight.io).
+
+[ajv]: https://www.npmjs.com/package/ajv
```

Comment:
We actually have not tested this markdown feature (versus just regular inline links) - if you open this up inside of studio, do the links work in the read panel? Do they work if you change one of these to be relative, e.g. `[get-started]: ./docs/getting-started/concepts.md`? By work I mean they render in the read panel, and when you click on it it takes you to the right external URL if absolute, or switches to the relevant local file if relative.

Reply:
I got Studio working and was able to check this out, it looks like [link reference definitions](https://spec.commonmark.org/0.29/#link-reference-definition) are not supported by our renderer, and/or do not show up in the read panel. 

<img width="1086" alt="Screen Shot 2019-08-06 at 11 30 10" src="https://user-images.githubusercontent.com/67381/62528728-9bec9600-b83d-11e9-935d-add9c99cf311.png">

@lottamus @P0lip think we could hook up the [CommonMark test cases](https://spec.commonmark.org/0.29/spec.json) to the Studio test suite, or our Markdown renderer, or something?

Reply:
Yeah we don't support definition yet: https://github.com/stoplightio/markdown-viewer/blob/master/src/utils/components.tsx#L212

There's a remark plugin we can add so they should be pretty easy to support: https://github.com/remarkjs/remark-inline-links. I'll open an issue to track

Reply:
Chris if you can add a bit of detail on generally where to start looking re implementation, perhaps @philsturgeon can take a crack at it - what do you think?

Reply:
new issue to track the work, dont worry @lottamus hope you got some sleep. https://github.com/stoplightio/studio-internal/issues/522